README.md

```diff
@@ -9,7 +9,7 @@
 [![npm](https://img.shields.io/npm/l/validx.svg?maxAge=1000)](https://github.com/jeffijoe/validx/blob/master/LICENSE.md)
 [![node](https://img.shields.io/node/v/validx.svg?maxAge=1000)](https://www.npmjs.com/package/validx)
 
-Validation library using MobX.
+Validation library with [MobX](https://github.com/mobxjs/mobx).
```

Comment:
Yeah I wasn't sold on using, with, or dependent on... It sounds too much like an addict.